### PR TITLE
Generate assets

### PR DESCRIPTION
**Pull request by melekr**
This diff contains the following updates:

- Distributes Backtrace-cocoa as XCFramework 

[assets](https://github.com/backtrace-labs/backtrace-cocoa/releases)

ref: BT-2721

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Backtrace Cocoa Release Notes
 
+## Version 2.0.5-beta2
```
Comment (konraddysput) [inline CHANGELOG.md:3]: we're adding more work for us in the future. Can we please continue adding more changes to the Version 2.0.5?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+- Generates static frameworks
+
+## Version 2.0.5-beta1
+- Adds BacktraceResources bundle
```
[the remark this answers is not in the document]
Comment (melekr) [reply CHANGELOG.md:7]: changelog is not up to date, will be updated soon.

File: CHANGELOG.md (hunk 1 above)
Comment (melekr) [reply CHANGELOG.md:3]: sure thing

File: scripts/deploy.sh [hunk 3]
```diff
@@ -6,4 +6,4 @@ set -o xtrace # to trace what gets executed. Useful for debugging
 
 # This script requires the `COCOAPODS_TRUNK_TOKEN` env var to be set. 
 # See more: https://fuller.li/posts/automated-cocoapods-releases-with-ci/.
-pod trunk push Backtrace.podspec --allow-warnings
+#pod trunk push Backtrace.podspec --allow-warnings
```
Comment (melekr) [inline scripts/deploy.sh:9]: disabled not to publish alpha pre-release

File: Backtrace.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -2537,12 +2546,24 @@
...
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "3rd Party Mac Developer Application";
+				"CODE_SIGN_IDENTITY[sdk=watchos*]" = "iPhone Distribution";
+				"CODE_SIGN_IDENTITY[sdk=xros*]" = "iPhone Distribution";
+				CODE_SIGN_STYLE = Manual;
 				COMBINE_HIDPI_IMAGES = YES;
 				COPY_PHASE_STRIP = NO;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_TEAM = "";
+				"DEVELOPMENT_TEAM[sdk=appletvos*]" = JWKXD469L2;
```
Comment (melekr) [inline Backtrace.xcodeproj/project.pbxproj:2561]: Github and xcodebuild are not playing well together.
I can create a custom build file to hide this.

File: Backtrace.podspec [hunk 5]
```diff
@@ -9,7 +9,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "Backtrace"
-  s.version      = "2.0.5-beta2"
+  s.version      = "2.0.5"
```
Comment (konraddysput) [inline Backtrace.podspec]: can you please rebase your branch with the latest master version?

**Review by konraddysput — CHANGES_REQUESTED** [review]
Everything looks fine - Not sure why did I read the old version of the pull request

**Review by konraddysput — APPROVED** [review]
Github messed up files in the pull request. It looks good to me ! Thanks!

**Pull request merged**